core/src/main/scala/kafka/log/ProducerStateManager.scala

```diff
@@ -445,8 +445,9 @@ class ProducerStateManager(val topicPartition: TopicPartition,
             lastMapOffset = lastSnapOffset
             return
           } catch {
-            case e: CorruptSnapshotException =>
-              error(s"Snapshot file at ${file.getPath} is corrupt: ${e.getMessage}")
+            case e: Exception =>
```

Comment:
I guess so.. Seems like `IOException`, `SchemaException`, and `CorruptSnapshotException`. We also throw `IllegalArgumentException` if the version is wrong, but I almost think `CorruptSnapshotException` is more appropriate for that case. Since we ignore these errors anyway, I thought we may as well just catch `Exception`.